kubernetes/zenko/charts/cloudserver/dashboards/cloudserver.json

```diff
@@ -1115,25 +1209,49 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "process_virtual_memory_bytes{instance=\"$pod\"}",
+          "expr": "avg(sum(abs(irate(cloud_server_http_requests_total{app=~\"cloudserver\"}[30m]))))",
           "format": "time_series",
-          "interval": "",
+          "hide": false,
           "intervalFactor": 1,
-          "legendFormat": "virtual",
+          "legendFormat": "Cumulative Requests",
           "refId": "A"
         },
         {
-          "expr": "process_resident_memory_bytes{instance=\"$pod\"}",
+          "expr": "avg(sum(abs(irate(cloud_server_number_of_ingested_objects{app=\"cloudserver\"}[30m]))))",
           "format": "time_series",
+          "hide": false,
           "intervalFactor": 1,
-          "legendFormat": "resident",
+          "legendFormat": "RClone Ingestion",
           "refId": "B"
+        },
+        {
+          "expr": "avg(sum(abs(irate(cloud_server_http_requests_total{method=\"HEAD\",app=~\"cloudserver\"}[30m]))))",
+          "format": "time_series",
+          "instant": false,
+          "intervalFactor": 1,
+          "legendFormat": "HEAD Requests",
+          "refId": "C"
+        },
+        {
+          "expr": "avg(sum(abs(irate(cloud_server_http_requests_total{method=\"PUT\",app=~\"cloudserver\"}[30m]))))",
+          "format": "time_series",
+          "instant": false,
+          "intervalFactor": 1,
+          "legendFormat": "PUT Requests",
+          "refId": "D"
+        },
+        {
+          "expr": "avg(sum(abs(irate(cloud_server_http_requests_total{method=\"GET\",app=~\"cloudserver\"}[30m]))))",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "GET Requests",
+          "refId": "E"
         }
       ],
       "thresholds": [],
       "timeFrom": null,
       "timeShift": null,
-      "title": "Memory",
+      "title": "PUT/GET/DELTE Operations",
```

Comment:
is it intentionally `DELTE`

Reply:
Oops. 1. Typo 2. It doesn't measure DELETE operations. Thank you